tests/test_order.py

```diff
@@ -160,14 +180,13 @@ def test_track_status_running(order_mock, requests_mock):
         },
     ]
     requests_mock.get(url_job_info, status_responses)
-    order_status = order_mock.track_status(report_time=0.1)
+    order_test = order.Order(constants.ORDER_ID)
```

Comment:
This was to fix an error in tests. Previously the test called the endpoint by the order_mock and the last of the status_responses, now the test is having the expected behavior.